core/src/main/java/com/linecorp/armeria/client/DefaultWebClient.java

```diff
@@ -61,7 +61,7 @@ public HttpResponse execute(HttpRequest req, RequestOptions requestOptions) {
         final RequestTarget reqTarget = RequestTarget.forClient(originalPath, prefix);
         if (reqTarget == null) {
             return abortRequestAndReturnFailureResponse(
-                    req, new IllegalArgumentException("Invalid path: " + originalPath));
+                    req, new IllegalArgumentException("Invalid request target: " + originalPath));
```

Comment:
This isn't always a path but can be an absolute URI, so I changed it to 'request target'.